src/QMCHamiltonians/EwaldRef.h

```diff
@@ -199,11 +199,12 @@ class KspaceEwaldTerm
 template<typename T>
 RealType gridSum(T& function,bool zero=true,RealType tol=1e-11)
 {
-  RealType dv = 1e99;
-  RealType v  = 0.0;
-  IntType im  = 0;
-  IntType jm  = 0;
-  IntType km  = 0;
+  RealType dv  = 1e99;
```

Comment:
if RealType is float, there will be a compiler warning. Use std::numeric_limits<RealType>::max Instead.

Reply:
Done.